lib/support/requests/anonymous/anonymous_contact.rb

```diff
@@ -33,6 +30,11 @@ def requester
             only_actionable.
             order("created_at desc")
         }
+
+        private
+        def anonymous_contact_params
```

Comment:
Yep. Thanks for the extra explanation. I got confused between controllers and lib given what I'd done for the previous Rails upgrades was just to replace `attr_accessible` with strong_params. This has now amended 903b010.